setup.py

```diff
@@ -1,27 +1,45 @@
-#!/usr/bin/env python
 from setuptools import setup, find_packages
 
-setup(name='django-oscar-api',
-      version='0.1',
-      url='https://github.com/tangentlabs/django-oscar-api',
-      author="David Winterbottom",
-      author_email="david.winterbottom@tangentlabs.co.uk",
-      description="REST API module for django-oscar",
-      long_description=open('README.rst').read(),
-      keywords="Oscar, REST, API",
-      license='BSD',
-      packages=find_packages(exclude=['sandbox*', 'tests*']),
-      include_package_data=True,
-      install_requires=[
-          'django-oscar>=0.4',
-      ],
-      # See http://pypi.python.org/pypi?%3Aaction=list_classifiers
-      classifiers=[
-          'Development Status :: 4 - Beta',
-          'Environment :: Web Environment',
-          'Framework :: Django',
-          'Intended Audience :: Developers',
-          'License :: OSI Approved :: BSD License',
-          'Operating System :: Unix',
-          'Programming Language :: Python']
-      )
+
+__version__ = "0.0.13"
+
+
+setup(
+    # package name in pypi
+    name='django-oscar-api',
+    # extract version from module.
+    version=__version__,
+    description="REST API module for django-oscar",
+    long_description=open('README.rst').read(),
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: Unix',
+        'Programming Language :: Python'
+    ],
+    keywords='',
+    author='Lars van de Kerkhof, Martijn Jacobs',
+    author_email='lars@permanentmarkers.nl, martijn@devopsconsulting.nl',
+    url='https://github.com/tangentlabs/django-oscar-api',
+    license='BSD',
+    # include all packages in the egg, except the test package.
+    packages=find_packages(exclude=['ez_setup', 'examples', 'tests', 'sandbox']),
+    # for avoiding conflict have one namespace for all apc related eggs.
+    namespace_packages=[],
+    # include non python files
+    include_package_data=True,
+    zip_safe=False,
+    # specify dependencies
+    install_requires=[
+        'setuptools',
+        'django-oscar',
```

Comment:
I'm not the one that could tell you that, since I'm only using oscar since version 0.9 or something. Then again, while it might be totally broken or some functionality might be broken on old versions, there have been many times that I wanted to use a package with an 'unsuited' version of a software and had to pull all kinds of tricks to work around such a specification. Existing projects on old version might not be easy to update, but might be able to work around some of the brokenness.

That said, If you could recommend some number, where it would be totally utterly broken and unfixable without overriding every last route, I am all ears.